src/style.css

```diff
@@ -0,0 +1,55 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-input {
+  height: 70px;
+  margin: 10px 30px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  padding: 25px 709px 23px 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  fill: #687c94;
+}
+
+.search-input:focus {
+  border-radius: 3px;
+  font-weight: 900;
+  fill: #687c94;
+}
+
+.search-input.search-input-small {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element